iOS/Flows/Conversation/Cells/ConversationMessageCell.swift

```diff
@@ -12,15 +12,15 @@ import UIKit
 
 /// A cell to display a high-level view of a conversation's message. Displays a limited number of recent replies to the message.
 /// The user's replies and other replies are put in two stacks (along the z-axis), with the most recent reply at the front (visually obscuring the others).
-class ConversationMessageCell: UICollectionViewCell {
+class ConversationMessageCell: UICollectionViewCell, ConversationMessageCellLayoutDelegate {
 
     // Interaction handling
     var handleTappedMessage: ((Messageable) -> Void)?
     var handleDeleteMessage: ((Messageable) -> Void)?
 
-    private let collectionView: UICollectionView = {
-        let layout = ConversationMessageCellLayout()
-        let cv = UICollectionView(frame: .zero, collectionViewLayout: layout)
+    private lazy var collectionLayout = ConversationMessageCellLayout(messageDelegate: self)
```

Comment:
The layout needs access to know which message is the latest one, so we pass in ourselves as a message delegate. More on this later.